doc-High_Availability_Guide/topics/HA_Roles.adoc

```diff
@@ -0,0 +1,25 @@
+[[ha_roles]]
+== Scaling a Highly Available {product-title_short} Environment
```

Comment:
Good idea @suyogsainkar  -- and now that I look at it again, it might be better worded/more consistent still as "Updating a Highly Available CloudForms Environment". I've changed the chapter title to that now.